masterlist.yaml

```diff
@@ -1641,14 +1641,19 @@ plugins:
         subs: [ 'Frostcrag Reborn LOD Toggler' ]
         condition: 'many("FRLODToggle(Basic|Full).esp")'
 
-  ### MigMaster Script Resources ###
+  - name: '(MigMaster|SafeCloningFunction - Filter)\.es[mp]'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/45875/'
+        name: 'MigMaster Script Resources'
   - name: 'MigMaster.esm'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/45875/' ]
     req: [ *OBSE ]
+    clean:
+      - crc: 0xE604721D
+        util: 'TES4Edit v4.0.4c'
   - name: 'SafeCloningFunction - Filter.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/45875/' ]
     group: *latePatchesGroup
     msg: [ *requiresBashedPatch ]
+    tag: [ -Deactivate ]
```

Comment:
Since the tag is just redundant, it doesn't need to be removed on our end. I imagine the removal could also confuse some users not familiar with Bash Tags or filter patches & make them think the plugin should be activated:

```suggestion
```